<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationWindowHome</class>
 <widget class="QWidget" name="InformationWindowHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,20,8,90">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="DateWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {

	background-color: rgb(18, 0, 0);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignBottom">
       <widget class="QLabel" name="Date">
        <property name="styleSheet">
         <string notr="true">QLabel {

font: bold 9pt  &quot;Heebo&quot;;
	color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);

}</string>
        </property>
        <property name="text">
         <string>SAMA SAMA TAYONG BABANGON MULI :REDHEART: :GREENHEART: :PEACE: :REDHEART: :GREENHEART:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="LogoWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
	border-top:2px solid rgb(213, 173, 18);
	background-image: url(:/Icons/ProgramIcons/Banner.png);

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,0,190">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel {

image: url(:/Icons/ProgramIcons/univlogo.png);

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QLabel" name="ButtonsWidget">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {

border-top:0px solid rgb(213, 173, 18);
font: bold  &quot;Heebo&quot;;
	color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);

}</string>
          </property>
          <property name="text">
           <string>TOM'S TECHNOLOGICAL UNIVERSITY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel {

border-top:0px solid rgb(213, 173, 18);
font:  &quot;Heebo&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);

}</string>
          </property>
          <property name="text">
           <string>WE BRING THE TECH TO YOU</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {

		
	background-color: rgb(18, 0, 0);
		border-top:2px solid rgb(213, 173, 18);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="20,20,20,20,20,20,200">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="HomeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{

font: bold 12pt &quot;Heebo&quot;;
background-color: rgba(13, 54, 101,0);
color: rgb(233, 236, 237);
padding: 5px;
}

QToolButton::hover{

		background-color: rgba(107, 4, 4,240);


}
QToolButton::pressed{
background-color: solid rgb(59, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="NewsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{

font: bold 12pt &quot;Heebo&quot;;
background-color: rgba(13, 54, 101,0);
color: rgb(233, 236, 237);
padding: 5px;
}

QToolButton::hover{

	background-color: rgba(107, 4, 4,240);


}
QToolButton::pressed{
background-color: solid rgb(59, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>News</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="AboutUsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{

font: bold 12pt &quot;Heebo&quot;;
background-color: rgba(13, 54, 101,0);
color: rgb(233, 236, 237);
padding: 5px;
}

QToolButton::hover{

	background-color: rgba(107, 4, 4,240);


}
QToolButton::pressed{
background-color: solid rgb(59, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>About Us</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ContactUsbutton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{

font: bold 12pt &quot;Heebo&quot;;
background-color: rgba(13, 54, 101,0);
color: rgb(233, 236, 237);
padding: 5px;
}

QToolButton::hover{

	background-color: rgba(107, 4, 4,240);


}
QToolButton::pressed{
background-color: solid rgb(59, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>ContactUs</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="FAQButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{

font: bold 12pt &quot;Heebo&quot;;
background-color: rgba(13, 54, 101,0);
color: rgb(233, 236, 237);
padding: 5px;
}

QToolButton::hover{

	background-color: rgba(107, 4, 4,240);


}
QToolButton::pressed{
background-color: solid rgb(59, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>FAQs</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="GalleryButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{

font: bold 12pt &quot;Heebo&quot;;
background-color: rgba(13, 54, 101,0);
color: rgb(233, 236, 237);
padding: 5px;
}

QToolButton::hover{

	background-color: rgba(107, 4, 4,240);


}
QToolButton::pressed{
background-color: solid rgb(59, 0, 0);
}

</string>
        </property>
        <property name="text">
         <string>Gallery</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="Contents">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
